run_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+IFS=$'\n'
+
+./a.out units/Tolvaj.json units/Robin.json >> results.txt
```

Comment:
Az elsőnél jobb lenne `>`-t használni, hogy ha már létezik a fájl, írja felül. És akkor lokálisan egymás után többször is futtatható.